lib/graphql/subscriptions.rb

```diff
@@ -202,26 +211,43 @@ def normalize_arguments(event_name, arg_owner, args)
           end
 
           if arg_defn
+            # TODO will this break compatibility with existing subscriptions?
```

Comment:
Gotta figure this out.

Reply:
I added the test for this behavior to master and it passed, so I think this will be fine.